.travis.yml

```
@@ -5,18 +5,20 @@ env:
  - DJANGO_VERSION=1.11
  - DJANGO_VERSION=2.0
  - DJANGO_VERSION=2.1
- - DJANGO_VERSION=2.2b
+ - DJANGO_VERSION=2.2
+ - DJANGO_VERSION=3.0
  - DJANGO_VERSION=master
 python:
  - "3.4"
  - "3.5"
  - "3.6"
  - "3.7"
+ - "3.8"
  - "pypy3.5"
 install:
  - if [[ $DJANGO_VERSION != master ]]; then pip install --pre -q "Django>=${DJANGO_VERSION},<${DJANGO_VERSION}.99"; fi
  - if [[ $DJANGO_VERSION == master ]]; then pip install https://github.com/django/django/archive/master.tar.gz; fi
- - pip install django-redis==4.10.0 flake8 python3-memcached>=1.51
+ - pip install django-redis>=4.10.0 flake8 python3-memcached>=1.51
```

might as well drop the `>=` marks here imo

```suggestion
 - pip install django-redis flake8 python3-memcached
```